src/widgets/configure/__tests__/configure-test.js

```diff
@@ -9,77 +8,4 @@ describe('configure', () => {
       () => configure(/ok/),
     ].forEach(widget => expect(widget).toThrowError(/Usage/));
   });
-
-  it('Applies searchParameters if nothing in configuration yet', () => {
-    const widget = configure({ analytics: true });
-    const config = widget.getConfiguration(SearchParameters.make({}));
-    expect(config).toEqual({
```

Comment:
did these tests still pass?

Reply:
Yes, just moved them to the connector side since it holds the logic now

Reply:
Yes but you can still test that the widget calls the appropriate functions of the connector.

Reply:
@samouss The widget shipped with instantsearch.js is a "dumb" one there's no logic nor rendering, only forwarding options to the connector. Does not makes senses.

Reply:
Yep I know but I prefer to always over tests rather than no tests, but I understood your point. I'm fine with it.